final_project/year_app/year_app.py

```diff
@@ -1,24 +1,67 @@
-from flask import Flask, render_template
+from flask import Flask, request, session, url_for, redirect, \
+     render_template, abort, g, flash, _app_ctx_stack
 import numpy as np
 import pickle
+import requests
+import pprint
+pp = pprint.PrettyPrinter(indent=4)
+
+from pyechonest import config
```

Comment:
Yea we can figure out best structure for the code later I was just trying to hack together a proof of concept.

> On Nov 19, 2015, at 1:42 PM, Max Wolffe <notifications@github.com> wrote:
> 
> In final_project/year_app/year_app.py <https://github.com/kcparashar/amadeus/pull/13#discussion_r45404763>:
> 
> >  import numpy as np
> >  import pickle
> > +import requests
> > +import pprint
> > +pp = pprint.PrettyPrinter(indent=4)
> > +
> > +from pyechonest import config
> Nvm. I see that you do. Maybe we can just move all of our echonest api requests into the echo_req.py file?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/kcparashar/amadeus/pull/13/files#r45404763>.
>